tools/dockerfile/distribtest/cpp_stretch_aarch64_cross_x64/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:stretch
```

Comment:
nit: current debian stable is "buster" so perhaps we wanna use that?

Reply:
That's a good point but it'd better have that aligned with other c++ tests which still rely on the same stretch. This might lead to another discussion about what version of debian we use with primarily. Stretch is the oldest of supported versions so we still need to have a test coverage there. We might have two tests per each? stretch (oldest supported) and buster (latest supported)?

Reply:
Ok, let's just keep stretch. Since the C++ distribtest are quite slow, I'd like to avoid duplication. Having tests that run for too long (without really providing a clear value) is very annoying.